src/storage/v2/inmemory/storage.cpp

```diff
@@ -1091,31 +1091,35 @@ InMemoryStorage::DropUniqueConstraint(LabelId label, const std::set<PropertyId>
 
 VerticesIterable InMemoryStorage::InMemoryAccessor::Vertices(LabelId label, View view) {
   auto *mem_label_index = static_cast<InMemoryLabelIndex *>(storage_->indices_.label_index_.get());
-  return VerticesIterable(mem_label_index->Vertices(label, view, &transaction_));
+  return VerticesIterable(mem_label_index->Vertices(label, view, &transaction_, &storage_->constraints_));
 }
 
 VerticesIterable InMemoryStorage::InMemoryAccessor::Vertices(LabelId label, PropertyId property, View view) {
   auto *mem_label_property_index =
       static_cast<InMemoryLabelPropertyIndex *>(storage_->indices_.label_property_index_.get());
-  return VerticesIterable(
-      mem_label_property_index->Vertices(label, property, std::nullopt, std::nullopt, view, &transaction_));
+  auto *mem_storage = static_cast<InMemoryStorage *>(storage_);
```

Comment:
I believe we can get rid of casting to InMemoryStorage to get constraints here and in more places in this file below.

Reply:
Yes I fixed this